public/images/shuttle-opt.svg

```diff
@@ -8124,7 +8124,10 @@
      fill-opacity="0"
      stroke="#000004"
      stroke-linecap="round"
-     stroke-width="0.265" />
+     stroke-width="0.265">
```

Comment:
The closing / was removed without adding a matching </path> tag. This broke the image making it unable to be rendered